tifeatures/layer.py

```diff
@@ -9,7 +9,7 @@
 from buildpg import funcs as pg_funcs
 from buildpg import logic, render
 from ciso8601 import parse_rfc3339
-from geojson_pydantic import Feature, FeatureCollection
+from geojson_pydantic import Feature, FeatureCollection  # type:ignore
```

Comment:
what is the issue here?

Reply:
🤔 https://github.com/developmentseed/geojson-pydantic/blob/master/geojson_pydantic/__init__.py#L17

can you check locally if you do `from geojson_pydantic.features import Feature, FeatureCollection`